railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt

```diff
@@ -13,6 +13,7 @@ Rails.application.configure do
   config.consider_all_requests_local = true
 
   # Enable/disable caching. By default caching is disabled.
+  # run rails dev:cache to toggle caching
```

Comment:
Two small changes on this line:
- can you capitalize run
- can you add a period at the end of the sentence